src/cli/cli.rs

```diff
@@ -0,0 +1,33 @@
+use configuration::Settings;
+use clap::Parser;
+use std::io::prelude::*;
+use std::fs::File;
+use std::path::Path;
+
+fn write_data_to_file() -> Result<(), serde::yaml::Error> {
+
+    // Assigns path to base.yaml file to variable. Immutable
+    let path = Path::new("./configuration/base.yaml")
+
+    // Insert code for variable holding default values
+    let settings = Settings::default();
+
+    // Insert code for serializing variable
+    let s = serde_yaml::to_string(&settings)?;
+
+    // Assigns the file to mutable variable
+    let mut buffer = File::create("base.yaml")?;
```

Comment:
what's the purpose of this variable? The `buffer` variable is actually never used.